appveyor.yml

```diff
@@ -1,7 +1,9 @@
 init:
   - git config --global core.autocrlf input
+install:
+  - cinst gtksharp -y
```

Comment:
Any reason we can't put this in build.cmd, and checkin the cinst.exe tool? Or is it too intrusive?